rust.html.markdown

```diff
@@ -256,11 +260,13 @@ fn main() {
     // `while` loop
     while 1 == 1 {
         println!("The universe is operating normally.");
+        break;
     }
 
     // Infinite loop
     loop {
         println!("Hello!");
+        break;
```

Comment:
I understand your changes are intended to make the code directly runnable by the Rust Playground. However, the addition of the `break` statement here rather defeats the purpose of the statement. Accordingly, could you please add a comment clarifying the situation?

Reply:
Yes. I will add commentary about using break.